compiler-rt/lib/builtins/cpu_model/aarch64/fmv/mrs.inc

```diff
@@ -45,8 +45,6 @@ static void __init_cpu_features_constructor(unsigned long hwcap,
     setCPUFeature(FEAT_SSBS2);
   if (hwcap2 & HWCAP2_MTE)
     setCPUFeature(FEAT_MEMTAG2);
-  if (hwcap2 & HWCAP2_MTE3)
```

Comment:
Do we need to remove runtime detection for these features for other targets as well?

Reply:
There are no uses of the removed features in other targets.